README.md

```diff
@@ -2,13 +2,24 @@
 
 A commandline utility to search across Slack for messages that were reacted to with a specific emoji.
 
-## Installation
+## Installation (simple)
 
-Ensure Python and virtualenv are set up. If you've run the [laptop script](https://github.com/18F/laptop), you're all good.
+The simplest way to install this package is to use [pip](https://pip.pypa.io/en/stable/installing/) to install the package from the [Python Package Index](https://https://pypi.python.org/pypi/slack-emoji-search):
```

Comment:
https://https://pypi.python.org/pypi/slack-emoji-search has an extra HTTPS in it :)